service-front/module/Application/view/layout/twig/accordion-layout.twig

```diff
@@ -14,11 +14,12 @@
     <ul class="accordion">
 
         {% set accordionTop = accordion( lpa ).top() %}
+        {% set route = routeName() %}
 
         {% include 'layout/partials/accordion/accordion.twig' with { 'items' : accordionTop, 'lpa':lpa } %}
 
-        {% if (accordionTop|last).routeName != 'review-link' %}
-            <li aria-hidden="true"><a id="current" class="accordion-current"></a></li>
+        {% if route['previous'] != 'user/about-you' and route['previous'] != 'user/dashboard' and (accordionTop|last).routeName != 'review-link' %}
```

Comment:
Does modifying `routeName()` have any effect on what gets set in `(accordionTop|last).routeName`? Does that need to be modified so that it uses the `current` property from the routeName or is it unaffected?

Reply:
They're entirely separate. I can't remember the class right now, but when I was initially looking at this it's a property of that class. 

Although we could just replace that now & use `routeName()`, but I thought it was a bit of, a can of worms. With how the track the path when working with the LPA itself.